<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QWidget" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="MyGLWidget" name="myGLWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>449</width>
           <height>280</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>xRot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="rotXSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>yRot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="rotYSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>zRot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="rotZSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>00</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>02</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>03</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>04</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>05</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>06</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>07</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>08</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rotXSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotYSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotZSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
